docs/ldml/tr35-keyboards.md

```diff
@@ -496,10 +496,27 @@ Element containing informative properties about the layout, for displaying in us
 >
 > Children: _none_
 >
-> Occurrence: optional, single
+> Occurrence: required, single
 >
 > </small>
 
+_Attribute:_ `name` (required)
+
+> This is the only required attribute.
+>
+> This name is not localized, but is an informative name for the keyboard.
```

Comment:
When you say not localized, does that mean it is in the primary locale of the keyboard? Or English? Or arbitrary locale of choice of the author? 

Not sure why there is _but_ here.

Reply:
I'll take out 'but', and 'not localized' doesn't add a lot here.  It was meant to say that there weren't multiple names per language for the keyboard

Reply:
I think you raised a good point. If not giving guidance to authors, we should at least have agreed understanding of what do we expect people to put here in terms of language.

Reply:
it could say 'this name may be in the primary language of the keyboard' or something?  `Bulgarian (Phonetic Traditional)` isn't a great example then.